test/servico/cst24Test.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package servico;
+
+import auxiliar.Perfil;
+import datamapper.PopulateDB;
+import datamapper.exceptions.NonexistentEntityException;
+import java.text.ParseException;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+import modelo.UsuarioModel;
+
+/*
+ *
+ * @author Mohamad
+ *
+ */
+
+public class cst24Test {
+    
+    @BeforeClass
+    public static void setUpClass() throws NonexistentEntityException, Exception{
+        PopulateDB.recreateDB("prosub", "root", "");
+        PopulateDB.populateUseCaseTest();
+    }
+    
+    //@AfterClass
+    //public static void tearDownClass() {
+    //}
+    
+    //@Before
+    //public void setUp() throws ParseException, NonexistentEntityException {
+    //}
+    
+    //@After
+    //public void tearDown() {
+    //}
+    
+    @Test
+    public void testeEdicaoDeUsuarioCancelada() throws ParseException, Exception{
+        
+        //Login como usuário "Administrador" 
+        LoginService loginService = new LoginService();
+        assertTrue(loginService.VerificarUsuarioESenha("Administrador", "123456"));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String "123456" appears 3 times in the file.](https://www.codacy.com/app/calebepb/substituicao/file/2777678509/issues/source?bid=3364462&fileBranchId=3364482#l52)